lib/pbench/server/api/resources/intake_base.py

```diff
@@ -277,18 +279,18 @@ def _intake(
                 )
                 try:
                     Dataset.query(resource_id=intake.md5)
-                except DatasetNotFound:
+                except DatasetNotFound as e:
                     raise APIInternalError(
-                        f"Duplicate dataset {dataset_name} for user {username!r} id {user_id} not found"
-                    )
+                        f"Duplicate dataset {intake.md5!r} ({dataset_name!r} is missing"
```

Comment:
You're missing a right-parenthesis after `{dataset_name!r}`.

Reply:
That's my penalty for changing the error message. Dang.